packages/opus/tangram.ts

```diff
@@ -0,0 +1,41 @@
+import * as std from "std" with { local: "../std" };
+import { $ } from "std" with { local: "../std" };
+import * as cmake from "cmake" with { local: "../cmake" };
+
+export const metadata = {
+	homepage: "https://opus-codec.org/",
+	name: "opus",
+	version: "1.5.2",
+	provides: {},
+};
+
+export const source = () => {
+	std.download;
+	const { name, version } = metadata;
+	const checksum =
+		"sha256:65c1d2f78b9f2fb20082c38cbe47c951ad5839345876e46941612ee87f9a7ce1";
+	return std
+		.download({
+			url: `https://downloads.xiph.org/releases/${name}/${name}-${version}.tar.gz`,
+			checksum,
+			mode: "extract",
+		})
+		.then(tg.Directory.expect)
+		.then((directory) => directory.get(`${name}-${version}`))
+		.then(tg.Directory.expect);
+};
+
+export type Arg = cmake.Arg;
+
+export const build = async (...args: std.Args<Arg>) => {
+	return cmake.build(
+		{ source: source() },
+		...args,
+	);
+};
+
+export const env = () => std.env.arg({
```

Comment:
For this issue, I found I could use `CMAKE_INSTALL_LIBDIR` - e.g. from `libuv`:

```
	const configure = {
		args: [
			"-DCMAKE_BUILD_TYPE=Release",
			"-DCMAKE_INSTALL_LIBDIR=lib",
			"-DBUILD_TESTING=OFF",
		],
	};
```

How do you feel about that instead?